minvws/nl-kat-coordination
# Add rocky worker service to debian packages

### Changes

This step should add the rocky worker service to the Debian packages as well.

### Issue link

-

### Demo

-
### QA notes

We should test the report generation flow on a Debian installation. Edit: @dekkers offered help in the comments below with testing the Debian packages and production docker setup if needed :pray: 

---

### Code Checklist




---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
